sds_data_manager/utils/stackbuilder.py

```diff
@@ -104,10 +106,30 @@ def build_sds(scope: App, env: Environment, account_config: dict):
     efs = efs_stack.EFSStack(scope, "EFSStack", networking.vpc, env=env)
 
     instrument_list = ["Codice"]  # etc
-
     lambda_code_directory = Path(__file__).parent.parent / "lambda_code" / "SDSCode"
     lambda_code_directory_str = str(lambda_code_directory.resolve())
 
+    spin_table_code = lambda_code_directory / "spin_table_api.py"
+    # Create Lambda for universal spin table API
+    spin_spin_api_handler = api_gateway_stack.APILambda(
```

Comment:
```suggestion
    spin_api_stack = api_gateway_stack.APILambda(
```

All the other routes live in the sds_data_manager stack. Should this be moved up there too? It kind of sticks out as the only Lambda at this level right now. But maybe we can reorganize again in a follow-up PR...

Reply:
I think future PR because I would like to group API lambda out. But I wanted to wait until other PR is merged.